File: E4.c
#include<stdio.h>
void main(){
    int i,j=0,temp[20],arr[20],n,count=0,stat;
    printf("Enter the limit : ");
    scanf("%d",&n);
    printf("Enter the array : ");
    for(i=0;i<n;i++){
        scanf("%d",&arr[i]);
    }
    for(i=0;i<n;i++){
        for(j=i+1;j<n;j++){
            if(arr[i]>arr[j]){
            stat=arr[i];
            arr[i]=arr[j];
            arr[j]=stat;
            }
        }
    }
    for(i=0;i<n;i++){
        printf("%d ",arr[i]);
    }
    for(i=0;i<n;i++){
        if(arr[i]!=arr[i+1]){
            temp[count++]=arr[i];
        }
    }
    for(i=0;i<n;i++){
        printf("%d",temp[i]);
    }
}

File: E4.java
import java.util.Scanner;
public class E4 {
    public static void main(String[] args) {
        Scanner sc= new Scanner(System.in);
        int i,j=0,stat;
        int[] arr=new int[50];
         System.out.println("Enter the limit : ");
         int len=sc.nextInt();
         System.out.println("Enter the array : ");
         for(i=0;i<len;i++){
             arr[i]=sc.nextInt();
         }
         for(int m=0;m<len;m++){
             for(int l=m+1;l<len;l++){
                 if(arr[m]>arr[l]){
                 stat=arr[m];
                 arr[m]=arr[l];
                 arr[l]=stat;
                 }
             }
         }
        for(i=0;i<len-1;i++){
            if(arr[i]!=arr[i+1]){
                arr[j++]=arr[i];
            }
        }
        arr[j++]=arr[len-1];
        for(int k=0;k<j;k++){
            System.out.print(" "+arr[k]);
        }
    }
}